index.html

```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Alzubair Protfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="header">
+    <header class="container">
+      <a class="navbar" href="">My Logo</a>
+      <nav class="button"><i class="fa-solid fa-bars"></i></nav>
+    </header>
+  </div>
+
+  <div class="section-2">
+    <img class="first-img" src="Microverse-Student-Project-1/img.png" alt="background-img">
+    <div class="paragraph">
+      <p class="first-text">I’m Alzubair Glad to see you!</p>
+      <p class="second-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="connect-button" href="#">LET’S CONNECT</a>
```

Comment:
- [ ] Kindly use a <h2....h6> tag for this title instead of an anchor tag. As this is a heading and not a link or button.